.readthedocs.yml

```diff
@@ -2,3 +2,9 @@ version: 2
 
 sphinx:
   configuration: doc/readthedocs/sphinx/conf.py
+
+# read the docs image build already includes sphinx-rtd-theme, so this isn't needed
+# however, keeping it here and running it on RTD is a near zero cost operation and makes sure the requirements are ok
+python:
+  install:
+    - requirements: doc/readthedocs/requirements.txt
```

Comment:
Comment explains this, but basically, if you are building the sphinx docs as a dev, install the reqs.  RTD doesn't actually need this, but it doesn't hurt anything.